Sources/YCoreUI/Extensions/UIKit/UIColor+rgbComponents.swift

```diff
@@ -10,7 +10,7 @@ import UIKit
 
 // This relatively simple tuple (four float values representing the color channels)
 // is already a released public api.
-// swiftlint: disable large_tuple
+// swiftlint:disable superfluous_disable_command large_tuple
```

Comment:
```suggestion
```
If it is superfluous in the latest SwiftLint (SwiftLint has gone back and forth on this large_tuple rule recently), then just delete this line.

Reply:
OK, so the `large_tuple` suppression is not superfluous, but the `superfluous_disable_command` is. Please review. Do you need to update your local SwiftLint?